packages/mitex/specs/prelude.typ

```diff
@@ -210,7 +253,12 @@
     spec.insert(key, value.at(0))
     if value.at(1) != none {
```

Comment:
There are many `value.at(0)` and `value.at(1)`, could be destruct `value` with nice names?